docs/develop/go/test-suites.mdx

```diff
@@ -178,7 +276,7 @@ Calling `s.env.ExecuteWorkflow(...)` executes the Workflow logic and any invoked
 
 > Note that unless the Activity invocations are mocked or Activity implementation replaced (see [Activity mocking and overriding](#mock-and-override-activities)), the test environment will execute the actual Activity code including any calls to outside services.
 
-After executing the Workflow in the above example, we assert that the Workflow ran through completion via the call to `s.env.IsWorkflowComplete()`. We also assert that no errors were returned by asserting on the return value of `s.env.GetWorkflowError()`.
+After executing the Workflow in the above example, we assert that the Workflow ran through completion via the call to `s.env.IsWorkflowCompleted()`. We also assert that no errors were returned by asserting on the return value of `s.env.GetWorkflowError()`.
```

Comment:
I think it's the code that is doing the assertions rather than "We" (which the style guide doesn't like). The test asserts that the Workflow ran through completion, using the call to `s.env.IsWorkflowCompleted()`, and that no errors were returned (`s.NoError(s.env.GetWorkflowError())`).

Consider moving this to the previous section since it describes code that lives there, and perhaps talking about in testing the Workflow, all assertions regarding proper operation must pass. Once passed, if the Workflow returned a value, you could retrieve it by calling `s.env.GetWorkflowResult....` in this section.

Reply:
This text is not by me. I'm just fixing the typo there: `IsWorkflowComplete` -> `IsWorkflowCompleted`.

Reply:
👍 Marking resolved.